ng-dev/release/stamping/env-stamp.ts

```diff
@@ -83,7 +86,8 @@ function getSCMVersions(
       const {version: experimentalVersion} = createExperimentalSemver(new SemVer(version));
       return {version, experimentalVersion};
     }
-  } catch {
+  } catch (e) {
+    console.error(e);
```

Comment:
will that work with stamping? or will Bazel "swallow" the message? should we just throw as well?

Reply:
My instinct is to say that we should throw here, but I am not sure if I missing something with respect to what this could break if we throw.  Or rather what it could break unintentionally.

Reply:
I think it would fail the build? (can be easily tested). If that"s the case, I'd propose going with that for better erroring/stamping

Reply:
Yeah, removing the try catch results in a good error message even from within bazel.

```
ng-dev release build-env-stamp

Build the environment stamping information

Options:
  --version  Show version number                                                                               [boolean]
  --help     Show help                                                                                         [boolean]
  --mode     Whether the env-stamp should be built for a snapshot or release [required] [choices: "snapshot", "release"]

TypeError: Invalid Version: zone.js-0.11.5
```